### PR TITLE
Added bulk submission page

### PR DESCRIPTION
**Pull request by abourne1**
new bulk submission page!

Screen Shot 2019-06-24 at 11 07 12 AM
Screen Shot 2019-06-24 at 2 02 10 PM

I added a "Change Password" page and bulk submission page to the docs. Also added an "info" box that displays after successfully changing your password

### CODE REVIEW

File: src/scss/style.scss [hunk 1]
```diff
@@ -3,4 +3,9 @@
 .error {
   @extend .text-primary-red;
   @extend .border-primary-red;
+}
+
+.info {
```
Comment (abourne1) [inline src/scss/style.scss:8]: @devowhippit these colors correspond to the `bg-green-100` and `border-green-700` tailwind classes, but it didn't look like I was able to use these classes directly? Are they not imported by the nyco patterns? 

Is what I've done here ok, or would it be better to add these tailwind colors to nyco patterns and then import them into screening docs?

**Review by KISS — COMMENTED** [review]
The `New Password` field should have the red asterisk as well since it's required. 

Other than this, LGTM.

**Comment by abourne1** [comment]
> The `New Password` field should have the red asterisk as well since it's required.
> 
> Other than this, LGTM.

thanks! asterick has been added

**Comment by devowhippit** [comment]
Awesome! Looks great :)

**Pull request merged**